docs/7_1/components/panelgrid.md

```diff
@@ -23,7 +23,9 @@ rendered | true | Boolean | Boolean value to specify the rendering of the compon
 binding | null | Object | An el expression that maps to a server side UIComponent instance in a backing bean
 columns | 0 | Integer | Number of columns in grid.
 style | null | String | Inline style of the panel.
+styleContent | null | String | Inline style of the panel-content.
 styleClass | null | String | Style class of the panel.
+styleClassContent | null | String | Style class of the panel-content.
 columnClasses | null | String | Comma separated list of column style classes.<br/>For layout=grid: Grid CSS - classes<br/>For layout=flex: PrimeFlex (FlexGrid) - classes; primeflex.css must be included into the template.xhtml
```

Comment:
In other cases like in autoComplete: https://primefaces.github.io/primefaces/7_0/#/components/autocomplete the classes are called "inputStyle,inputStyleClass" and "panelStyle, panelStyleClass" so I think the right names for consistency would be "contentStyle" and "contentStyleClass".

Reply:
Good point.